crux-test/src/crux/fixtures.clj

```diff
@@ -22,43 +70,21 @@
           [:crux.tx/put m ts]))))
 
 (defn transact!
-  "Helper fn for transacting entities"
+  "Helper fn for transacting entities "
   ([api entities]
    (transact! api entities (cio/next-monotonic-date)))
   ([^ICruxAPI api entities ts]
-   (doto (api/submit-tx api (maps->tx-ops entities ts))
-     (->> (api/await-tx api)))
+   (doto (crux/submit-tx api (maps->tx-ops entities ts))
+     (->> (crux/await-tx api)))
    entities))
 
-(defn entities->delete-tx-ops [entities ts]
-  (vec (for [e entities]
-         [:crux.tx/delete e ts])))
-
-(defn delete-entities!
-  ([api entities]
-   (delete-entities! api entities (cio/next-monotonic-date)))
-  ([api entities ts]
-   (let [submitted-tx (api/submit-tx api (entities->delete-tx-ops entities ts))]
-     (api/await-tx api submitted-tx))
-   entities))
-
-(defn random-person [] {:crux.db/id (UUID/randomUUID)
-                        :name      (rand-nth ["Ivan" "Petr" "Sergei" "Oleg" "Yuri" "Dmitry" "Fedor" "Denis"])
-                        :last-name (rand-nth ["Ivanov" "Petrov" "Sidorov" "Kovalev" "Kuznetsov" "Voronoi"])
-                        :sex       (rand-nth [:male :female])
-                        :age       (rand-int 100)
-                        :salary    (rand-int 100000)})
+(defn random-person []
+  {:crux.db/id (UUID/randomUUID)
+   :name (rand-nth ["Ivan" "Petr" "Sergei" "Oleg" "Yuri" "Dmitry" "Fedor" "Denis"])
+   :last-name (rand-nth ["Ivanov" "Petrov" "Sidorov" "Kovalev" "Kuznetsov" "Voronoi"])
+   :sex (rand-nth [:male :female])
+   :age (rand-int 100)
+   :salary (rand-int 100000)})
```

Comment:
At some stage we should look to see if `random-person` and `transact!` are stilling pulling their weight, if not, kill.

Reply:
agreed :)